src/pages/soknad/soknaden.tsx

```diff
@@ -76,6 +77,12 @@ const Fordeling = () => {
         case RSSoknadstatus.UTKAST_TIL_KORRIGERING:
             return (
                 <>
+                    <Vis hvis={valgtSoknad.status === RSSoknadstatus.UTKAST_TIL_KORRIGERING}>
+                        <AlertStripe type='info' className='blokk-s'>
+                            <span>Rediger det som er feil i søknaden, og send den inn på nytt.</span>
```

Comment:
Burde ikke dette ligge i en av tekster-filene? Og er `<span>` nødvendig for at det skal se riktig ut?

Reply:
Trenger span for å få det til å se riktig ut, jeg kan flytte teksten inn i tekster